### ISSUE TITLE
hey sudeep manasali

### ISSUE THREAD

**Issue by pannagkumaar** [sudeepmanasali/form-builder-app#1]
hey i wanted to know about the google form clone i am following along on your youtube but i wasnt able to complete it please help me regarding that

**Comment by sudeepmanasali** [comment]
Yeah, I made it private because it's under development and I have changed
it for other purpose
I will search in my system and I will send you zip file tommorow

**Comment by pannagkumaar** [comment]
Okay thank you for responding was facing issues when typing the code
because there were gaps and some parts of the code which wasn't visible
please send the code the earliest

**Comment by pannagkumaar** [comment]
I have followed along till the 8th video on your channel and wasnt ablet to
use the drag and drop feature and all thee other features
You dont have to send be the entire project file but please do send the
files from components folder

**Comment by pannagkumaar** [comment]
hello Sudeepmanasali/Form-Builder-App please send the codes for
Google-forms-clone from the components folder

**Comment by pannagkumaar** [comment]
heyy please respond , i have been trying to rectify using your videos but i am not able to at all there are too many issues popping up please help me!

**Comment by pannagkumaar** [comment]
HELLO Sudeepmanasali please respond it has been 3 days since you said youll
be sending the codes please respond i am unable to complete the project

**Comment by sudeepmanasali** [comment]
https://drive.google.com/file/d/1H4D1f46n5YQVEkcU0qUOKlrLAnI70LSr/view?usp=sharing

**Comment by pannagkumaar** [comment]
hello can you please give me access to access the files im unable to do so

**Comment by sher-vishu** [comment]
Hi Sudeep! I was following your videos on Google form clone. I have downloaded the zip file you provided here for the changed version of Google form. But I found that neither did it have the backend files nor did it have the node modules. Could you please upload the whole folder for google form clone? It would be a great help to check where I am going wrong.
